src/cli.js

```diff
@@ -32,7 +34,7 @@ let cli = {
       // Show help
       console.log(options.generateHelp());
     } else {
-      nibbler.setExtensions(extensions);
+      nibbler.setExtensions(extensions, config);
```

Comment:
I think I've updated this pull request with your requested changes. Let me know if there's anything else I should be fixing.